src/main/java/edu/escuelaing/ieti/matchmaking/model/Activity.java

```diff
@@ -2,16 +2,29 @@
 
 import java.sql.Date;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "typ")
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = GroupActivity.class, name = "GroupActivity"),
+        @JsonSubTypes.Type(value = IndividualActivity.class, name = "IndividualActivity")})
 public abstract class Activity  {
 
+    @JsonFormat(pattern="yyyy-MM-dd'T'HH:mm:ss")
     private Date date;
+    @JsonFormat(pattern="yyyy-MM-dd'T'HH:mm:ss")
```

Comment:
You can annotate every field this way or you can customize your marshaller to handle every Date the same way. Try something like this in your configuration class:

`@Bean
public Jackson2ObjectMapperBuilder objectMapperBuilder() {
    Jackson2ObjectMapperBuilder builder = new Jackson2ObjectMapperBuilder();
    builder. simpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
    return builder;
}`